Bottom: b9fcbbe22233d2007bdcd1968ea5577e91d1f1ef
Top:    d8f958ad235a44c2af19425b9144c5d00ca15386
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-06 16:47:09 +0000

Refresh of more-play-with-quad-edge-data

---

diff --git a/src/hid/common/edge3d.c b/src/hid/common/edge3d.c
index d2ae858..d95260f 100644
--- a/src/hid/common/edge3d.c
+++ b/src/hid/common/edge3d.c
@@ -9,8 +9,6 @@ make_edge_info (void)
   edge_info *info;
 
   info = g_new0 (edge_info, 1);
-  info->is_stitch = is_stitch;
-  info->is_round = is_round;
 
   return info;
 }
